### PR TITLE
Enable add counters into energy configuration in HA

### PR DESCRIPTION
**Pull request by AlexObukhoff**
I try to add my saures gas counter from sauresha to HA energy config.
But button "Add gas source" not showing my sauresha gas sensor. 
By my opinion its not working because unit_of_measurement is `м3 `, but need `m3`. But its not enough.

### CODE REVIEW

File: custom_components/sauresha/sensor.py [hunk 1]
```diff
@@ -220,15 +220,24 @@ def fetch_state(self):
                     'approve_dt': meter.approve_dt,
                 })
             if self.isStart:
-                if meter.type_number == 1 or meter.type_number == 2 or meter.type_number == 3:
+                if meter.type_number == 1 or meter.type_number == 2:
                     self._attributes.update({
-                        'unit_of_measurement': 'м³'})
+                        'unit_of_measurement': 'm³',
+                        'device_class': 'water',
```
Comment (AlexObukhoff) [inline custom_components/sauresha/sensor.py:226]: Wrong device_class

**Comment by AlexObukhoff** [comment]
Wrong device_class 
see #24

File: custom_components/sauresha/sensor.py (hunk 1 above)
Comment (vsimanin) [reply custom_components/sauresha/sensor.py:226]: not work!

Comment (AlexObukhoff) [reply custom_components/sauresha/sensor.py:226]: Правильно. Поэтому я этот реквест закрыл, и создал #24 
@volshebniks уже его втащил в мастер.

**Pull request closed without merging**